Resources/Prototypes/Entities/Clothing/Shoes/boots.yml

```diff
@@ -55,6 +55,23 @@
   - type: Clothing
     sprite: Clothing/Shoes/Boots/combatboots.rsi
 
+- type: entity
+  parent: ClothingShoesMilitaryBase
+  id: ClothingShoesHighheelBoots
+  name: high-heeled boots
+  description: Snazy boots for when you want to be stylish, yet prepared.
+  components:
+  - type: MeleeWeapon
+    damage:
+      types:
+        Slash: 12
```

Comment:
remove or turn down because 12 slash is actually a pretty good melee weapon

Reply:
Ahh good catch, meant to make it pierce anyway.

Reply:
Changed to 6 piercing (same as a screwdriver)